lms/templates/admin/macros.html.jinja2

```diff
@@ -52,10 +52,10 @@
         {% endif %}
     {% endcall %}
 {% endmacro %}
-{% macro settings_textarea(object, label, setting, sub_setting, field_name, default='') %}
+{% macro settings_textarea(object, label, setting, field_name, default='') %}
```

Comment:
These only two uses of `settings_textarea`, base them passing the full setting object.